x-pack/plugins/security_solution/public/network/containers/tls/index.tsx

```diff
@@ -94,79 +84,55 @@ export const useNetworkTls = ({
     [limit]
   );
 
-  const [networkTlsResponse, setNetworkTlsResponse] = useState<NetworkTlsArgs>({
-    tls: [],
-    id: ID,
-    inspect: {
-      dsl: [],
-      response: [],
+  const {
+    loading,
+    result: response,
+    search,
+    refetch,
+    inspect,
+  } = useSearchStrategy<NetworkQueries.tls>({
+    factoryQueryType: NetworkQueries.tls,
+    initialResult: {
+      edges: [],
+      totalCount: -1,
```

Comment:
I know this is what was there before but find it a little odd. It's not related to your PR, but i think we should set this to 0. and change code elsewhere that would be problematic because of that. 

In public/common/components/paginated_table/index.tsx we have
`  const pageCount = Math.ceil(totalCount / limit);` 
which is why I think we have a `-1` as default as otherwise, if it were 0 this calculation would equal Infinity. But we could change the code here to be `totalCount ? totalCount / limit : 1, or some such equivolent. 

Maybe there is another reason though

Reply:
++ to the suggestion above. At least we need some explanation in the code why it is -1.

Reply:
https://github.com/elastic/kibana/issues/136672